fdbrpc/FlowTransport.actor.cpp

```diff
@@ -324,13 +324,16 @@ struct Peer : NonCopyable {
 			pkt.canonicalRemotePort = transport->localAddresses.secondaryAddress.get().port;
 			pkt.setCanonicalRemoteIp(transport->localAddresses.secondaryAddress.get().ip);
 		} else {
-			pkt.canonicalRemotePort = 0;   // a "mixed" TLS/non-TLS connection is like a client/server connection - there's no way to reverse it
+			pkt.canonicalRemotePort =
+			    0; // a "mixed" TLS/non-TLS connection is like a client/server connection - there's no way to reverse it
```

Comment:
This seems like a weird bit of auto-formatting.

Reply:
yes, I will fix this.